src/main/java/com/nyxei/pollingus/utils/HtmlBuilder.java

```diff
@@ -0,0 +1,68 @@
+package com.nyxei.pollingus.utils;
+
+import com.nyxei.pollingus.models.Poll;
+import com.nyxei.pollingus.models.enums.AnswerType;
+import com.nyxei.pollingus.models.question.Proposition;
+import com.nyxei.pollingus.models.question.Question;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.stream.IntStream;
+
+public class HtmlBuilder {
+    public static String format(Poll poll){
+        StringBuilder builder = new StringBuilder();
+        builder.append("<h1>").append(poll.getName()).append("</h1>");
+        builder.append("<h2>").append(poll.getTopic()).append("</h2>");
+        IntStream.range(0, poll.getQuestions().size())
+                .forEach(index -> {
+                    poll.getQuestions().get(index).setNumber(index + 1);
+                    builder.append(HtmlBuilder.format(poll.getQuestions().get(index)));
+                });
+        return builder.toString();
+    }
+
+    public static String format(Question question){
+        StringBuilder builder = new StringBuilder("<h5>").append(StringUtils.defaultString(String.valueOf(question.getNumber()), "#"));
+        builder.append(") ").append(question.getFormulation()).append("</h5>");
+        switch (question.getAnswerType()){
+            case DATE:
+                builder.append("<input type='date' name='question").append(question.getId()).append("'>");
```

Comment:
Maybe you should prevent the case where parameters values could change or being injected, there's even the possibility to internationalize these values.